hazelcast-enterprise/Dockerfile

```diff
@@ -14,13 +14,10 @@ ARG HAZELCAST_ALL_URL="https://repository.hazelcast.com/release/com/hazelcast/ha
 
 # Runtime variables
 ENV HZ_HOME="${HZ_HOME}" \
+    USER_NAME="hazelcast" \
```

Comment:
Inline `USER_NAME` to move to `ARG`. I guess you don't need it at runtime.

Reply:
good catch 👍